aries-site/src/pages/templates/forms.mdx

```diff
@@ -216,11 +200,60 @@ For standard forms, use an asterisk(\*) to indicate required field/controls.
   <SingleFormFieldExample />
 </Example>
 
-## Customize
+### Customize
 
 <Example
   code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/forms/CustomizeExample.js"
   figma="https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=54%3A608"
 >
   <CustomizeExample />
 </Example>
+
+### Character counter
+
+**When to use**  
+In cases where the user is given an opportunity to express themselves or would likely exceed a field's character limit. Examples include some description fields, feedback, feature requests, reporting issues, etc.
+
+**Guidance**
+
+- Place the character counter at the bottom right of the field.
+- [See guidance on character limits for fields](https://design-system.hpe.design/components/tag#anatomy). Note that your backend may have character limitations. When you can choose the character limits, however, set a generous number to avoid frustrating the user. A suggestion to ensure your limit is user-friendly, check the analytics to see how long it is taking your users to complete this task. Longer than usual time could indicate the character limits are too low. Increase accordingly.
```

Comment:
- Internal links should not include the `https://design-system.hpe.design`.
- Hyperlinked portion should typically be focused/limited to the subject of the sentence.
- Link does not seem like it is pointing to the correct content.

```suggestion
- See guidance on [character limits for fields](/components/tag#anatomy). Note that your backend may have character limitations. When you can choose the character limits, however, set a generous number to avoid frustrating the user. A suggestion to ensure your limit is user-friendly, check the analytics to see how long it is taking your users to complete this task. Longer than usual time could indicate the character limits are too low. Increase accordingly.
```